### ISSUE TITLE
Return flights combined with one-way flights

### ISSUE THREAD

**Issue by montenegroraabe** [amabnl/amadeus-ws-client#290]
Hi!

One question: the only way to search for round trip flights is to do two itineraries? 
`'itinerary' => [
	        new MPItinerary([
	            'departureLocation' => new MPLocation(['city' => 'GRU']),
	            'arrivalLocation' => new MPLocation(['city' => 'BEY']),
	            'date' => new MPDate([
	                'dateTime' => new \DateTime('2019-03-05T00:00:00+0000', new \DateTimeZone('UTC'))
	            ])
	        ]),
	        new MPItinerary([
	            'departureLocation' => new MPLocation(['city' => 'BEY']),
	            'arrivalLocation' => new MPLocation(['city' => 'GRU']),
	            'date' => new MPDate([
	                'dateTime' => new \DateTime('2019-03-09T00:00:00+0000', new \DateTimeZone('UTC'))
	            ])
	        ]) 
	    ],`

So, according to the Master Price examples, I make two calls on MPItinerary but the flights are separated, one-way flights in one array and flights back in another array. Is there a simple call that brings the combined flights or only this way?

**Comment by rifkydj** [comment]
hi, as i know there is no way to get combined flight in one array. you have to reference them by the key.  #235 . please check amadeus xml logs too.

**Comment by DerMika** [comment]
Yes, you need to add two itineraries in your masterpricer call for a roundtrip. 

And the result is indeed a very complicated structure, as far as I know, there is nothing available from Amadeus to make this response easier to use.

**Comment by humanyu** [comment]
Can anyone tell me how to add airline class like economy, business, first etc. In fare_masterpricetravelboardsearch

**Comment by DerMika** [comment]
Closing due to lack of feedback, you can re-open if needed!
